atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServices.java

```diff
@@ -89,23 +92,23 @@ static void waitForSchemaVersions(
         while (System.currentTimeMillis() < start + schemaTimeoutMillis);
 
         StringBuilder sb = new StringBuilder();
-        sb.append(String.format("Cassandra cluster cannot come to agreement on schema versions,"
-                + " after attempting to modify table %s.", tableName));
+        String messageTemplate = "Cassandra cluster cannot come to agreement on schema versions,"
+                + " after attempting to modify table {}. {}  "
```

Comment:
I think this requires a new line since we only prepend new lines in the string builder.